README.md

```diff
@@ -4,10 +4,31 @@ This is a repository for CLUE curriculum unit and teacher guide data. The unit a
 
 ## Adding a Unit
 
-To add a new unit, first decide on a code for the unit. For example, the existing unit Stretching and Shrinking uses the code `sas`. Next, create a subfolder using that code as a name. In that new subdirectory, add a file named `[unit-code]-unit.json`. The file name for Stretching and Shrinking, for example, is sas-unit.json.
+To add a new unit, first decide on a unit code. For example, the existing unit _Stretching and Shrinking_ uses the code `sas`. Next, create a new subdirectory in the curriculum directory using that code as a name. In that new subdirectory, add a file named `content.json`. Use the unit-template.json file as a starting place to build the new file's content.
 
-Use the unit-template.json file as a starting place to build the new unit file's content.
+### Problem Sections
+Curriculum units contain a number of investigations. Each investigation contains a number of problems, and each problem has a number of sections. The content for these problem sections is located in files separate from the unit's main content.json file.
+
+Each problem section should have its own content.json file located in a subdirectory of the unit directory. The file structure is generally:
+
+-- unit
+---- content.json
+---- investigation-1
+------ problem-1
+-------- section-1-type
+---------- content.json
+-------- section-2-type
+---------- content.json
+------ problem-2
+-------- section-1-type
+---------- content.json
+
+Note that each problem section's directory name should match the section's type. So if the problem section's type is `introduction`, its directory name should be `introduction`.
+
+Use the problem-section-template.json file as a starting place to build a problem section's content.json file. To populate the `tiles` array with the tiles that should appear in the section, first create the tiles in CLUE, then use CLUE's export/copy-to-clipboard feature to gather the tile's JSON, and then add it to the the `tiles` array.
```

Comment:
I'd include the keystrokes to export here: cmd+option+e for a single tile, cmd+shift+s to copy the whole document, cmd+option+shift+s to export the whole document as sections.

Reply:
Good idea. Done.